src/CoreWCF.WebHttp/src/CoreWCF/UriTemplatePathSegment.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using System.Text;
+using CoreWCF.Runtime;
+
+namespace CoreWCF
+{
+    // This represents a Path segment, which can either be a Literal, a Variable or a Compound
+    [DebuggerDisplay("Segment={originalSegment} Nature={nature}")]
+    internal abstract class UriTemplatePathSegment
```

Comment:
Fix DebuggerDisplay variable names to match new names

Reply:
Addressed in 14ac88e67a220d805d2fac57fa31026cd397ba1e.